api_factory/schema/wrappers.py

```diff
@@ -105,14 +105,17 @@ def __getattr__(self, name):
         return getattr(self.method_pb, name)
 
     @property
-    def overloads(self):
-        """Return the overloads defined for this method."""
-        return self.method_pb.options.Extensions[overload_pb2.overloads]
+    def field_headers(self) -> Sequence[str]:
+        """Return the field headers defined for this method."""
+        http = self.options.Extensions[annotations_pb2.http]
+        if http.get:
+            return tuple(re.findall(r'\{([a-z][\w\d_.]+)=', http.get))
```

Comment:
This RE is capturing the initial brace and the ending equals but does not strip them. I think this was not caught because this method is being implicitly tested through the `has_field_headers` test but not explicitly tested.

Reply:
The capture group does not include the brace.

Reply:
Ack.

Reply:
To be clear, because I understand why this is non-obvious:

```
    Return a list of all non-overlapping matches in the string.
    
    If one or more capturing groups are present in the pattern, return
    a list of groups; this will be a list of tuples if the pattern
    has more than one group.
```